phac-nml/irida-next
# Group Advanced Settings - Change Path

## What does this PR do and why?
Updated the group settings page to separate updates to path and parent namespace within the advanced settings section.
Fixes the first half of #75.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/b5e3e28f-bd77-4dd1-a44e-5d4b764a62e6)

## How to set up and validate locally

How to test the new form:
1. Navigate to the new group page
2. Enter a group name.
3. Verify path is slugified.
4. Hit the `Create group` button.
5. Verify the group was created.

How to test the edit form:
1. Navigate to a group settings page.
2. Update path.
3. Hit the `Change path` button.
4. Verify the path changed. 

Note: It might be a good idea to test both groups and subgroups. 

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
